semantic/src/site/globals/site.overrides

```diff
@@ -77,6 +77,11 @@ a:hover {
   justify-content: space-between;
 }
 
+.flex-row-space_evenly {
```

Comment:
I think for consistency it would be better called `flex-row-space-evenly` and elsewhere too

Reply:
Yup, done.